README.md

```diff
@@ -650,6 +652,25 @@ version 'v1', using: :param, parameter: 'v'
     curl http://localhost:9292/statuses/public_timeline?v=v1
 
 
+## Linting
+
+You can check if your API is in conformance with [Rack's specification](https://github.com/rack/rack/blob/main/SPEC.rdoc) by calling `lint!` at the API level or through [configuration](#configuration):
```

Comment:
```suggestion
You can check whether your API is in conformance with the [Rack specification](https://github.com/rack/rack/blob/main/SPEC.rdoc) by calling `lint!` at the API level or through [configuration](#configuration).
```

Let's just break up these to make it look like the rest of the doc? People tend to copy paste things and will copy paste those together if we leave them together. And we can remove the comments because it's obvious that the last two are "through configuration".

```
class Api < Grape::API
  lint!
end
```

```
Grape.configure do |config|
  config.lint = true
end
```

```
Grape.config.lint = true
```